package.json

```diff
@@ -3,6 +3,12 @@
   "version": "1.9.0",
   "author": "Tobias Koppers @sokra",
   "description": "Offers a dev middleware for webpack, which arguments a live bundle to a directory",
+  "keywords": [
+    "html",
```

Comment:
From those keywords, only `webpack` is relevant. It has nothing to do with html, server-side or "advance-api" (I don't know what that is)?